src/test/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/IssueDocumentHandlerTest.java

```diff
@@ -195,4 +199,24 @@ public void givenHearingIsNull_thenReturnDraftDecisionNotice() {
         assertEquals(expectedDefaultDocumentLabel, documentTypeLabel);
     }
 
+    @Test
+    public void givenPostHearingReviewIsCorrection_thenUseCorrectionBody() {
+        SscsCaseData sscsCaseData = buildCaseData();
+        sscsCaseData.getPostHearing().setReviewType(PostHearingReviewType.CORRECTION);
+        String bodyContent = "correction body content";
+        sscsCaseData.getDocumentGeneration().setCorrectionBodyContent(bodyContent);
+
+        NoticeIssuedTemplateBody payload = handler.createPayload(null, sscsCaseData, "doctype", LocalDate.now(), LocalDate.now(), false, USER_AUTHORISATION);
+
+        assertEquals(payload.getNoticeBody(), bodyContent);
+    }
+
+    @Test
+    public void givenPostHearingReviewIsLibertyToApply_thenThrowException() {
```

Comment:
I would change this to ParameterizedTest and call it givenPostHearingReviewTypeIsNotImplemented_thenThrowException. Make the input PostHearingReviewType.class with the names of the three yet to be implemented.